extension/src/experiments/model/quickPick.test.ts

```diff
@@ -147,7 +147,7 @@ describe('pickExperiments', () => {
           description: '[exp-456]',
           detail: `Created:${formatDate(
             mockedExperiments[1].Created as string
-          )}, split:2.2000e+9, data/data.xml:22a1a29`,
+          )}, split:2200043556, data/data.xml:22a1a29`,
```

Comment:
[Q] Can we increase the size of this number so that it goes back to scientific notation for the test?